recipes/gdk-pixbuf/all/conanfile.py

```diff
@@ -79,117 +63,119 @@ def validate(self):
-
-            with open_temp_file("CMakeLists.txt") as cmake_file:
-                cmake_file.write(r"""
-                    cmake_minimum_required(VERSION 3.16)
-                    project(compiler_rt_test)
-                    try_compile(HAS_COMPILER_RT ${CMAKE_BINARY_DIR} ${CMAKE_SOURCE_DIR}/test.c OUTPUT_VARIABLE OUTPUT)
-                    if(NOT HAS_COMPILER_RT)
-                    message(FATAL_ERROR compiler-rt not present)
-                    endif()""")
-            with open_temp_file("test.c") as test_source:
-                test_source.write(r"""
-                    extern __int128_t __muloti4(__int128_t a, __int128_t b, int* overflow);
-                    int main() {
-                        __int128_t a;
-                        __int128_t b;
-                        int overflow;
-                        __muloti4(a, b, &overflow);
-                        return 0;
-                    }""")
-            cmake.definitions["CMAKE_EXE_LINKER_FLAGS"] = "-rtlib=compiler-rt"
-            try:
-                cmake.configure(source_folder=tmp)
-            except ConanException as ex:
-                raise ConanInvalidConfiguration("LLVM Compiler RT is required to link gdk-pixbuf in debug mode") from ex
-
     @functools.lru_cache(1)
     def _configure_meson(self):
         meson = Meson(self)
-        defs = {}
-        if scm.Version(self.version) >= "2.42.0":
-            defs["introspection"] = "false"
-        else:
-            defs["gir"] = "false"
-        defs["docs"] = "false"
-        defs["man"] = "false"
-        defs["installed_tests"] = "false"
-        if scm.Version(self.version) >= "2.42.8":
-            defs["png"] = "enabled" if self.options.with_libpng else "disabled"
-            defs["tiff"] = "enabled" if self.options.with_libtiff else "disabled"
-            defs["jpeg"] = "enabled" if self.options.with_libjpeg else "disabled"
-        else:
-            defs["png"] = "true" if self.options.with_libpng else "false"
-            defs["tiff"] = "true" if self.options.with_libtiff else "false"
-            defs["jpeg"] = "true" if self.options.with_libjpeg else "false"
-
-        defs["builtin_loaders"] = "all"
-        defs["gio_sniffing"] = "false"
-        defs["introspection"] = "enabled" if self.options.with_introspection else "disabled"
-        args = []
-        # Workaround for https://bugs.llvm.org/show_bug.cgi?id=16404
-        # Ony really for the purporses of building on CCI - end users can
-        # workaround this by appropriately setting global linker flags in their profile
-        if self._requires_compiler_rt:
-            args.append('-Dc_link_args="-rtlib=compiler-rt"')
-        args.append("--wrap-mode=nofallback")
-        meson.configure(defs=defs, build_folder=self._build_subfolder, source_folder=self._source_subfolder, pkg_config_paths=".", args=args)
+        meson.configure()
         return meson
```

Comment:
The `_configure_meson` method should no longer be necessary when using the new MesonToolchain generator.